### DISCUSSION TITLE
pdb reader

### DISCUSSION THREAD

**Discussion by Elaine2028** [community/community#118770] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Does anyone know a pdb reader? I found Cool Reader but it doesn't work with my Android ver. 14. Isilo is also incompatible. I have a whole library of pdb books. Please help...

**Comment by Akash1134** [comment]
Hi @Elaine2028 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Elaine2028** [comment]
It's just that I saw you are the developer of Cool Reader, which is a pdb
reader...
